### ISSUE TITLE
Decide what to do with TestTendermintTC7

### ISSUE THREAD

**Issue by piersy** [clearmatics/autonity#488]
# Problem

The test has been unconditionally skipped since it is too long to run on CI. In its current form it provides no value to the project.

# Acceptance Criteria

Decide what action to take with TestTendermintTC7

**Comment by JekaMas** [comment]
Maybe we can remove it completely after @Jason-Zhangxin-Chen manages to add python tests to the CI?

**Comment by yazzaoui** [comment]
I always run the full test suite including TestTendermintTC7 locally, so it's not like this test has no value. The historical reason : TestTendermintTC7 is the integrated version of an e2e test that @Jason-Zhangxin-Chen wrote for his framework. It was failing from time to time a few months ago when we were running long-duration tests overnight because of a race condition. So after I integrated this test I managed to replicate the issue but it was a very rare race condition, hence the time it's taking to complete.
 
More generally, I think there is high value in those integrated long duration tests, the problem is that it's not very practical to have them running on the CI for each PR, each time restarting after pushing a commit. One idea is to have them running only once the PR got merged on develop..
